tools/README-generator/make_readme.py

```diff
@@ -1,54 +1,65 @@
-#! /usr/bin/python3
+#! /usr/bin/env python3
 
 import argparse
 import json
 import os
+from pathlib import Path
 
 from jinja2 import Environment, FileSystemLoader
 
 
-def generate_READMEs(app_path):
+def generate_READMEs(app_path: Path):
 
-    if not os.path.exists(app_path):
+    if not app_path.exists():
         raise Exception("App path provided doesn't exists ?!")
 
-    manifest = json.load(open(os.path.join(app_path, "manifest.json")))
+    manifest = json.load(open(app_path / "manifest.json"))
```

Comment:
omfg i thought we could only do this in the context of pytest

This is fucking awesome, I've been dreaming of this :star_struck: